### ISSUE TITLE
EASE grid static assumptions in plotting

### ISSUE THREAD

**Issue by JimCircadian** [icenet-ai/icenet#109]
* IceNet version: 0.2.2

### Description

The data extents are hardcoded from a baseline figure of `data_extent_estimate = 5400000.0` within `icenet.plotting.utils`

This is accurate based on `(432 / 2) * 25000` for the limits of the data transform when using cartopy, but should be dynamic. I'm not concerned with dealing with this just yet, but it's a debt that should be dealt with. We also need to judge whether or not OSISAF's EASE grid (defined by the following snippet) is the only grid we should consider (for the moment, it is)

```
        data_globe = ccrs.Globe(datum="WGS84",
                                inverse_flattening=298.257223563,
                                semimajor_axis=6378137.0)
```

**Comment by bnubald** [comment]
@JimCircadian Could this be something that needs consideration when using AMSR2 data source?

**Comment by JimCircadian** [comment]
> @JimCircadian Could this be something that needs consideration when using AMSR2 data source?

Yes, if we do incorporate AMSR then masks, projections and linear trends will all be affected. 👍🏼
